pkg/agent/netpol/netpol.go

```diff
@@ -126,6 +129,9 @@ func Run(ctx context.Context, nodeConfig *config.Node) error {
 	wg.Add(1)
 	go hc.RunCheck(healthCh, stopCh, &wg)
 
+	//set klog v2 verbosity level
+	flag.Set("v", strconv.Itoa(cmds.LogConfig.VLevel))
+
```

Comment:
Drop this and just let the goroutine handle it
```suggestion
```

Reply:
We may miss some logs when kube-router is initialized but it's probably not a big deal.